app/src/test/java/com/claraberriel/mvpparking/mvp/presenter/ReservationPresenterTest.java

```diff
@@ -0,0 +1,288 @@
+package com.claraberriel.mvpparking.mvp.presenter;
+
+import com.claraberriel.mvpparking.entities.Parking;
+import com.claraberriel.mvpparking.entities.Reservation;
+import com.claraberriel.mvpparking.mvp.model.ReservationModel;
+import com.claraberriel.mvpparking.mvp.view.ReservationView;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.robolectric.RobolectricTestRunner;
+
+import java.util.Calendar;
+import java.util.Date;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(RobolectricTestRunner.class)
+public class ReservationPresenterTest {
+    ReservationPresenter presenter;
+    @Mock
+    ReservationModel model;
+    @Mock
+    ReservationView view;
+
+    @Before
+    public void setUp() {
+        MockitoAnnotations.openMocks(this);
+        presenter = new ReservationPresenter(view, model);
+    }
+
+    /**
+     * Event Handlers: success cases
+     */
+
+    @Test
+    public void onFromClicked_showDialog(){
+        presenter.onFromClicked();
+
+        verify(view).startDateTimeDialog();
+    }
+
+    @Test
+    public void onToClicked_showDialog(){
+        presenter.onToClicked();
+
+        verify(view).endDateTimeDialog();
+    }
+
+    @Test
+    public void onSchedule_isTrue(){
```

Comment:
if I read this test name, I don't understand what is being tested here. Please think about using more realistic names for the methods so the code is easier to read and understand